src/main/java/com/playground/playground/MlParametersController.java

```diff
@@ -0,0 +1,50 @@
+package com.playground.playground;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.text.Text;
+
+import java.net.URL;
+import java.util.Objects;
+import java.util.ResourceBundle;
+
+public class MlParametersController implements Initializable {
+    @FXML
+    private Button stepButton;
+
+    @FXML
+    private Button playButton;
+    @FXML
+    private Button rewindButton;
+    @FXML
+    private Text epochNumber;
+
+    /**
+     * Initializer for MIParametersController.java
+     * @param location
+     * The location used to resolve relative paths for the root object, or
+     * {@code null} if the location is not known.
+     *
+     * @param resources
+     * The resources used to localize the root object, or {@code null} if
+     * the root object was not localized.
+     */
+    public void initialize(URL location, ResourceBundle resources) {
+        // setting buttons
+        setButtonWithImage(rewindButton, "playground-images/rewind-button.png");
```

Comment:
We shouldn't do PNGs at all, I would always recommend in decreasing order of recommendation:

1. Use icons from some icons library or native icons (almost always or always do this), if you have a new icon (which we don't) import SVGs to these icons
2. Use .svg directly or .webp
3. Use some other vector graphics forms like .eps
4. .apng or .avif but only for lossless animation sequences
5. .png
6. .jpeg (please never do this)